### ISSUE TITLE
[Feature Request] Start directory for Load Configuration

### ISSUE THREAD

**Issue by jayare5** [libretro/RetroArch#14378]
## Curent behavior
We have 1 option for this in Directory:
```
Configs:
Set the start directory for menu configuration browser
```

But that description is not really all it does, it does 2 things at the same:
- Sets where the core overrides/configs/shaders are located.
- Sets as a start directory for the Load Configuration menu.

## Expected behavior
Would like to separate those things into 2 different Directory settings:
```
Core Configs:
Sets where the core overrides/configs/shaders are located.

RetroArch Configs:
Sets as a start directory for the Load Configuration menu.
```

This means I could have a folder with multiple RetroArch configs neatly named and organized in a folder, and not have to go looking for it every time I go into Load Configuration and in starts in the core configs folder.

**Comment by Ryunam** [comment]
Wouldn't this be addressed by simply creating a subfolder right inside the core "config" directory, where you could store the RA .cfg files that you want to load at any given time? It does require an additional click every time you select Load Configuration and I understand that it's slightly inconvenient this way, but perhaps it can serve as a good enough workaround.

**Comment by jayare5** [comment]
@Ryunam 
Heh, I do something similar already, and I have streamlined it as much as possible, yet it's still messy.
Remember! If you have the core config folder with a subfolder containing retroarch configs... but the retroarch config you load leads to a different core config folder, that would make Load configuration start you at this OTHER config folder that does not contain the subfolder with retroarch configs, and you'd have to navigate out of there and into the right one... unless you copy paste this sub folder, which means maintaining all copies... yeah.

You either have to take a bunch of steps to go to a folder with all the cfg files, or take a bunch of steps to navigate out of the folder and into another folder full of core configs and a retroarch.cfg.

**Comment by Ryunam** [comment]
Ah, I see what you mean! If you have multiple RetroArch configs and they all end up pointing to different subfolders for the core config directory, that will make it messy for sure...

**Comment by jayare5** [comment]
@Ryunam 
Yup, my solution was to make sub folders just outside the core configs, but that's still a little messy, because it involves backing out a few times, then going in some directory. That was the best I could come up with. But after a few years of this it's tiring, sometimes when it's been a while I forget Im supposed to navigate out first xD then I put a folder in there with a name that reminds me to go the upper folders first.
